app/forms/location_form.rb

```diff
@@ -152,7 +152,8 @@ def generate_names(prefix, start_from, end_to)
 
   def create_locations(params)
     locations = []
-    prefix = params[:location][:name]
+    # Remove double spaces and any interloping unfriendly characters
+    prefix = params[:location][:name].gsub('  ', ' ').strip
```

Comment:
Looks good. Would it be worth doing a regex for 2 or more spaces, in case we get the same in future for 3 spaces? :)